doc/Build_Environment.rst

```diff
@@ -0,0 +1,367 @@
+|**Name**                |**Description**                                                                            |
++========================+===========================================================================================+
+|Motr                    |Git clone ssh:/gitosis@git.clusterstor.com/motr.git                                        |
+|                        |                                                                                           |
+|                        | Root repository holding the following:                                                    |
+|                        |                                                                                           |
+|                        |   - Motr-core - Motr repository                                                           |
+|                        |                                                                                           |
+|                        |   - Motr-db4 - modified db4 repository                                                    |
+|                        |                                                                                           |
+|                        |   - Motr-get - bash script for fetching both repositories                                 |
+|                        |                                                                                           |
++------------------------+-------------------------------------------------------------------------------------------+
+|Motr-core               |Git clone ssh:/gitosis@git.clusterstor.com/motr-core.git                                   |
+|                        |                                                                                           |
+|                        | Repository for storing motr-core source code.                                             |
++------------------------+-------------------------------------------------------------------------------------------+
+|Motr-db4                | Git clone ssh:/gitosis@git.clusterstor.com/motr-db4.git                                   |
++------------------------+-------------------------------------------------------------------------------------------+
+
+As one can see, motr-core and motr-db4 are separate repositories. There is no some logical explanation why we need them separately now as there are no requirements dictating this structure. We surely can have such a requirements in future and more flexible repository structure allows to adopt quickly. Additionally, there are not many drawbacks in current structure to not do so.
+
+While motr-db4 repository is having completely the same structure as standard db4 distribution (we do not want to multiply differences, so that we could switch to new version quickly), motr-core, on the other hands, having the structure, discussed below.
```

Comment:
Now, please update this table with our CURRENT git configurations.
and remove "motr-db4" related sentences and paragraphes.


```
cortx-motr: https://github.com/Seagate/cortx-motr
cortx-motr-galois: https://github.com/Seagate/cortx-motr-galois
```